ideascube/serveradmin/catalog.py

```diff
@@ -126,6 +129,10 @@ def remove(self, package):
         self._removed.append(package)
 
     def commit(self):
+        visible_packages = get_config('ideascube', 'displayed-package-ids')
+        [visible_packages.remove(p.id) for p in self._removed if p.id in visible_packages]
```

Comment:
> Nope, it would not work, because visible_packages contains package ids, not package objects.
That explaining why I didn't right it at first time :p

What about this ? :

```python
removed_ids = {p.id for p in self._removed}
visible_packages = [id for id in visible_packages if id not in removed_ids]
```